network-api/networkapi/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -297,3 +300,73 @@ def test_category_page_context_with_cta_enabled(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertIsNotNone(response.context['featured_cta'])
+
+
+class BuyersGuideCallToActionTest(TestCase):
```

Comment:
Have you considered creating a new test module for this class? The model definition does live in a separate module than the `homepage.py`. The other test modules are named after the modules they are testing. E.g. `test_utils.py` tests functionality from `utils.py`. In this case it would mean to move this test class to a module `test_call_to_action.py`.

Reply:
Hi Tibor! This makes sense, I have created test_call_to_action.py!